CONTRIBUTING.md

```diff
@@ -81,14 +83,30 @@ For the versioning all packages in this repository are following the semantic ve
 
 ## Creating Changesets
 
-We use [changesets](https://github.com/atlassian/changesets) to help us prepare releases. They help us make sure that every package affected by a change gets a proper version number and an entry in its `CHANGELOG.md`. To make the process of generating releases easy, it helps when contributors include changesets with their pull requests. These changesets can be created by going into a `workspace` directory and running `yarn changeset` and following the prompts.
+We use [changesets](https://github.com/atlassian/changesets) to help us prepare releases. They help us make sure that every package affected by a change gets a proper version number and an entry in its `CHANGELOG.md`. To make the process of generating releases easy, it helps when contributors include changesets with their pull requests.
+
+To create a changeset, follow these steps:
+
+1. Make sure you are in the root directory of the workspace for the plugin you want to create a changeset for. For ex: if you are making changes on the `adr` plugin then you should be on `workspaces/adr` dir
+
++2. Run the following command to create a new changeset:
 
 ```bash
-cd community-plugins
-cd workspaces/linguist
-yarn changeset # follow the prompts
+$ yarn changeset
 ```
 
+3. You will be prompted to select the packages and the type of change you are making.
+
+4. Enter a short description of the change when prompted.
```

Comment:
we could also put a link to this section https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#writing-changesets